tests/readers/test_facebook_reader.py

```diff
@@ -15,88 +15,246 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program; if not, write to the Free Software Foundation,
 # Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
 from unittest import TestCase, mock
-from freezegun import freeze_time
+from parameterized import parameterized
+from click import ClickException
 
-from nck.readers.facebook_reader import FacebookMarketingReader
+from nck.readers.facebook_reader import FacebookReader
 
 from facebook_business.api import FacebookAdsApi
 from facebook_business.adobjects.adsinsights import AdsInsights
+from facebook_business.adobjects.ad import Ad
 
 
 class FacebookReaderTest(TestCase):
-    DATEFORMAT = "%Y-%m-%d"
 
-    def mock_facebook_reader(self, **kwargs):
-        for param, value in kwargs.items():
-            setattr(self, param, value)
+    DATEFORMAT = "%Y-%m-%d"
 
     kwargs = {
-        "ad_insights": True,
         "app_id": "",
         "app_secret": "",
-        "access_token": "",
-        "ad_object_ids": "123456789",
-        "recurse_level": 0,
-        "ad_object_type": "adaccount",
-        "desired_fields": ["date_start", "impressions"],
+        "access_token": "123456789",
+        "object_id": "123456789",
```

Comment:
Ici pour bien respecter le paramètre d'entrée du reader, il faut mettre:
```suggestion
        "object_id": ["123456789"],
```
Car c'est une liste en entrée